### ISSUE TITLE
been able to has a unique URL as well for "View Page"

### ISSUE THREAD

**Issue by efremfilho** [huynhsamha/dontpad#1]
### Expected behavior
Instead of the button only change the view, sends a parameter to the URL só we can share the page mode.

### Actual behavior
When clicking on "View Page", dynamically change to page view mode, but it doesn't generate a unique URL.

**Comment by huynhsamha** [comment]
@efremfilho : Thanks for your issue, I'll update the feature

**Comment by huynhsamha** [comment]
@efremfilho : I have updated the feature.

Now,  if the user enters the path with the suffix `view`, in case it is the first time the web page, ie the browser does not have the build React from the previous server returned, Node.JS will process redirect to the page without the suffix, then with this path, the Node.JS will send to the React build, and React will render the editor to the user, and connect the socket to the Node. In case the user has visited the site once, ie the browser has build React (SPA - Single Page Application), then React handle user navigation on the editor as above, Node.JS will not execute redirection as well as render anything in this case.

With the editor path, users can click on the `View Page` button, React will now redirect to the path containing the view suffix, and the user can view the View of this page.

In this update, I've added `react-router-dom` to handle the path in React for a SPA application, and also use Redux to store Props and State. It also updates other features such as User experience.
